### ISSUE TITLE
`capitalize` should regenerate `full_name`

### ISSUE THREAD

**Issue by twall** [derek73/python-nameparser#87]
```
obj = NameParser("UNILEVER LTD")
obj.full_name # "UNILEVER LTD"
obj.capitalize(force=True)
obj.full_name # "UNILEVER LTD" instead of "Unilever Ltd", even though other internal fields have been updated
```

**Comment by derek73** [comment]
fixed in ce92f379e96a324178e259a378aab628cbc12a6e

**Comment by derek73** [comment]
Thanks for the bug report. I think this was left over from when `full_name` was functioning more like a reference to the original name, but now there is `original_name` for that. Returning the current formatted string output makes more sense, even if it's just a reference to `str(obj)`.
